SignalServiceKit/src/Account/TSAccountManager.m

```diff
@@ -70,9 +71,9 @@ - (void)setPhoneNumberAwaitingVerification:(NSString *_Nullable)phoneNumberAwait
 {
     _phoneNumberAwaitingVerification = phoneNumberAwaitingVerification;
 
-    [[NSNotificationCenter defaultCenter] postNotificationName:kNSNotificationName_LocalNumberDidChange
-                                                        object:nil
-                                                      userInfo:nil];
+    [[NSNotificationCenter defaultCenter] postNotificationNameAsync:kNSNotificationName_LocalNumberDidChange
```

Comment:
This is used by `OWSSignalService` to update censorship circumvention. Safe to deliver later.